packages/flutter_tools/lib/src/commands/analyze_once.dart

```diff
@@ -183,20 +196,32 @@ class AnalyzeOnce extends AnalyzeBase {
 
     if (errorCount > 0) {
       // we consider any level of error to be an error exit (we don't report different levels)
-      if (membersMissingDocumentation > 0 && flutterRepo)
+      if (membersMissingDocumentation > 0)
         throwToolExit('[lint] $membersMissingDocumentation public ${ membersMissingDocumentation == 1 ? "member lacks" : "members lack" } documentation (ran in ${elapsed}s)');
       else
         throwToolExit('(Ran in ${elapsed}s)');
     }
     if (argResults['congratulate']) {
```

Comment:
Update the docs to mention that congratulate will be ignored if you're not using either `--watch` or `--flutter-repo`?